packages/cli/src/commands/versions/bump.ts

```diff
@@ -25,6 +25,8 @@ import chalk from 'chalk';
 import ora from 'ora';
 import semver from 'semver';
 import { OptionValues } from 'commander';
+import yaml from 'js-yaml';
```

Comment:
Use `yaml` here instead if possible, since that's what we already use in other places